proof-templates.json

```diff
@@ -967,6 +967,32 @@
                            "issuer_did": "RGjWbW1eycP7FrMf4QJvX8"
                         }
                      ],
+                     "devRestrictions": [
```

Comment:
Does Kyle and co. want dev restrictions for this proof? When I first made it he told me not to add any

Reply:
okay I'll remove it then